client/validators/profile.ts

```diff
@@ -85,3 +90,39 @@ export const formProfile = ({field, value, profile, errors, messages, diff}) =>
         delete errors[field];
     }
 };
+
+interface IValidateCustomCVArgs {
+    field: string;
+    value: undefined; // Value is `undefined`, as there is no field called `custom_vocabularies`
+    profile: IEditorProfile;
+    errors: {[field: string]: string};
+    messages: Array<string>;
+    diff: Partial<IEventOrPlanningItem>;
+}
+
+export function formProfileCustomVocabularies({field, value, profile, errors, messages, diff}: IValidateCustomCVArgs) {
+    const schema = profile.schema?.[field];
+
+    if (schema == null || schema.type !== 'list' || profile.editor[field]?.enabled !== true) {
+        return;
+    } else if (schema.required !== true) {
+        return;
+    }
+
+    planningApi.vocabularies.getCustomVocabularies()
+        .filter((cv) => schema.vocabularies?.includes(cv._id))
+        .forEach((cv) => {
+            const values = ((diff[cv.schema_field || 'subject'] || []) as Array<ISubject>).filter((item) => (
+                item.scheme === cv._id
+            ));
+            const fieldName = `custom_vocabularies.${cv._id}`;
+            const fieldLabel = gettext(cv.display_name).toUpperCase();
```

Comment:
only static strings should be passed to gettext. I think we don't support translating CV names at the moment and when we do support it should come from DB as it now works with translatable vocabulary values.